src/web/resources/report-page.js

```diff
@@ -848,11 +905,14 @@ async function fetchAndUpdate(jsonData) {
 
 async function getResultsJson() {
     if (resultsJsonData == null) {
-        let response = await fetch("results.json", {
-            headers: { "content-type": "text/plain" },
```

Comment:
Oops.

Reply:
I don't think the headers matter, I just removed them as cleanup.